client/components/account-balances/balance-tooltip.tsx

```diff
@@ -3,29 +3,147 @@
 
 /**
  * Internal dependencies
  */
 import { ClickTooltip } from 'components/tooltip';
+import { documentationUrls, fundLabelStrings } from './strings';
+import InlineNotice from 'components/inline-notice';
 
-type BalanceTooltipProps = {
-	label: string;
-	content: React.ReactNode;
+type TotalBalanceTooltipProps = {
+	balance: number;
 };
 
-const BalanceTooltip: React.FC< BalanceTooltipProps > = ( {
-	label,
-	content,
+type AvailableBalanceTooltipProps = {
+	balance: number;
+};
+
+export const TotalBalanceTooltip: React.FC< TotalBalanceTooltipProps > = ( {
+	balance,
 } ) => {
 	return (
 		<ClickTooltip
-			content={ content }
 			className="wcpay-account-balances__balances__item__tooltip"
 			buttonIcon={ <HelpOutlineIcon /> }
-			buttonLabel={ label }
+			buttonLabel={ `${ fundLabelStrings.total } tooltip` }
+			maxWidth={ '315px' } // So that tooltip is wide enough and the content in the inline notice is not wrapped.
+			content={
+				<>
+					<>
+						{ interpolateComponents( {
+							mixedString: __(
+								'{{bold}}Total balance{{/bold}} combines both pending funds (transactions under processing) and available funds (ready for deposit). {{learnMoreLink}}Learn more{{/learnMoreLink}}',
+								'woocommerce-payments'
+							),
+							components: {
+								bold: <b />,
+								learnMoreLink: (
+									// eslint-disable-next-line jsx-a11y/anchor-has-content
+									<a
+										rel="external noopener noreferrer"
+										target="_blank"
+										href={
+											documentationUrls.depositSchedule
+										}
+									/>
+								),
+							},
+						} ) }
+					</>
+					<InlineNotice
+						className="wcpay-account-balances__balances-total-balance-tooltip-notice"
+						isDismissible={ false }
+					>
+						{ __(
+							'Total balance = Available funds + Pending funds',
+							'woocommerce-payments'
+						) }
+					</InlineNotice>
+					<>
+						{ balance < 0 &&
+							interpolateComponents( {
+								mixedString: __(
+									'Negative account balance? {{discoverWhyLink}}Discover why.{{/discoverWhyLink}}',
+									'woocommerce-payments'
+								),
+								components: {
+									discoverWhyLink: (
+										// eslint-disable-next-line jsx-a11y/anchor-has-content
+										<a
+											rel="external noopener noreferrer"
+											target="_blank"
+											href={
+												documentationUrls.negativeBalance
+											}
+										/>
+									),
+								},
+							} ) }
+					</>
+				</>
+			}
 		/>
 	);
 };
 
-export default BalanceTooltip;
+export const AvailableBalanceTooltip: React.FC< AvailableBalanceTooltipProps > = ( {
+	balance,
+} ) => {
+	return (
+		<ClickTooltip
+			className="wcpay-account-balances__balances__item__tooltip"
+			buttonIcon={ <HelpOutlineIcon /> }
+			buttonLabel={ `${ fundLabelStrings.available } tooltip` }
+			maxWidth={ balance < 0 ? '280px' : undefined } // So that the negative balance sentence is not wrapped and looks like the design.
```

Comment:
Nitpick: Maybe to verbose to have this logic in the props, maybe we can extract it out?

Reply:
Address in [0b9ab72](https://github.com/Automattic/woocommerce-payments/pull/8140/commits/0b9ab7207c8cabd7052b187c1b503a915d7174dc).